app/models/ReviewByGoldenTicket.scala

```diff
@@ -104,15 +104,12 @@ object ReviewByGoldenTicket {
   }
 
   def allAllowedProposalsNotReviewed(reviewerUUID: String): List[Proposal] = Redis.pool.withClient {
-    return if (ConferenceDescriptor.isCFPOpen) {
-      ReviewByGoldenTicket.allProposalsNotReviewed(reviewerUUID)
-        .filterNot(p => p.talkType == ConferenceDescriptor.ConferenceProposalTypes.KEY || p.talkType == ConferenceDescriptor.ConferenceProposalTypes.OTHER)
-        .filterNot(_.sponsorTalk)
-    } else {
-      ReviewByGoldenTicket.allProposalsNotReviewed(reviewerUUID)
-        .filter(p => p.talkType == ConferenceDescriptor.ConferenceProposalTypes.CONF || p.talkType == ConferenceDescriptor.ConferenceProposalTypes.TIA || p.talkType == ConferenceDescriptor.ConferenceProposalTypes.LAB || p.talkType == ConferenceDescriptor.ConferenceProposalTypes.QUICK || p.talkType == ConferenceDescriptor.ConferenceProposalTypes.UNI)
-        .filterNot(_.sponsorTalk)
-    }
```

Comment:
@nicmarti I'm not 100% of the assumption I made about this simplification here, I mean :

isCFPOpen=true => black listing KEY and OTHER proposal types
isCFPOpen=false => white listing CONF / TIA / LAB / QUICK / UNI proposal types
implicitely: GT have never the right to vote on KEY & OTHER proposal types, and are not able to vote on BOF either once CFP is closed (this latest rule sounds weird to me)

If that was not the idea behind the if/else, don't hesitate to tell me :-)